tests/src/Regressions/coreclr/GitHub_7829/test7829.cs

```diff
@@ -0,0 +1,44 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+// Repro case for CoreCLR 7829
+
+class X
```

Comment:
We add API tests like this to CoreFX repo. The CoreFX repo has a better infrastructure for test like these.

The tests in CoreCLR repo are the core runtime tests (JIT, typesystem, interop, etc.).

Reply:
Ok. I'll move it over!

Reply:
Looks like there's already one there that's covering this so I'll just remove the duplicated test from this PR.